core/build.gradle

```diff
@@ -149,6 +150,7 @@ task integrationTestCassandra(type: Test) {
     options {
         systemProperties(System.getProperties())
     }
+    maxParallelForks = 10
```

Comment:
The first thing I did is setting `maxParallelForks` to run the integration tests concurrently.